shoreline/manifest.json

```diff
@@ -0,0 +1,76 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "90e1b0ed-0907-4973-929c-7e7f1be0c4f4",
+  "app_id": "shoreline-integration",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Category::Automation",
+    "Category::Marketplace",
+    "Category::Remediation",
+    "Category::Incident automation"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Troubleshooting",
+    "changelog": "CHANGELOG.md",
+    "description": "Build automations to repair triggered monitors - all from the Datadog UI.",
+    "title": "Shoreline Web Service Integration",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Remediation dashboard",
+        "image_url": "images/remediation_dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Example of remediation automation setup",
+        "image_url": "images/automate_remediation.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Example of fleetwide interactive debugging and repair",
+        "image_url": "images/fleetwide_interactive_debugging_and_repair.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Example of fleetwide linux command details",
+        "image_url": "images/fleetwide_linux_command_details.png"
+      }
+    ]
+  },
+  "author": {
+    "homepage": "https://shoreline.io/",
+    "name": "Shoreline Inc",
+    "sales_email": "sales@shoreline.io",
+    "support_email": "support@shoreline.io"
+  },
+  "oauth": {},
+  "assets": {
+    "dashboards": {},
+    "integration": {
+      "source_type_name": "Shoreline Web Service Integration",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "shoreline_web_service_integration.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    }
+  },
+  "legal_terms": {
+    "eula": "assets/terms_of_service_shoreline.pdf"
+  },
+  "pricing": [
+    {
+      "billing_type": "free"
```

Comment:
@mgarabed Thank you for your review! 

Could please help me with the pricing? I would like to add pricing but can't find a specification for it.

Reply:
Since this will be on the integrations page, no pricing object needed.